tests/ttnn/unit_tests/gtests/tensor/test_create_tensor.cpp

```diff
@@ -71,8 +76,38 @@ INSTANTIATE_TEST_SUITE_P(
     ::testing::Values(
         CreateTensorParams{.shape=ttnn::SimpleShape({1, 1, 32, 32})},
         CreateTensorParams{.shape=ttnn::SimpleShape({2, 1, 32, 32})},
+        CreateTensorParams{.shape=ttnn::SimpleShape({64, 1, 256, 1})},
+        CreateTensorParams{.shape=ttnn::SimpleShape({1, 1, 21120, 16})},
         CreateTensorParams{.shape=ttnn::SimpleShape({0, 0, 0, 0})},
         CreateTensorParams{.shape=ttnn::SimpleShape({0, 1, 32, 32})},
         CreateTensorParams{.shape=ttnn::SimpleShape({0})}
     )
 );
+
+class EmptyTensorTest : public ttnn::TTNNFixtureWithDevice, public ::testing::WithParamInterface<EmptyTensorParams> {};
+
+TEST_P(EmptyTensorTest, Tile) {
+    EmptyTensorParams params = GetParam();
+    auto tensor = tt::tt_metal::create_device_tensor(params.shape, DataType::BFLOAT16, ttnn::TILE_LAYOUT, device_, ttnn::DRAM_MEMORY_CONFIG);
+    EXPECT_EQ(tensor.get_logical_shape(), params.shape.logical_shape());
+}
+
+TEST_P(EmptyTensorTest, RowMajor) {
+    EmptyTensorParams params = GetParam();
+    auto tensor = tt::tt_metal::create_device_tensor(params.shape, DataType::BFLOAT16, ttnn::ROW_MAJOR_LAYOUT, device_, ttnn::DRAM_MEMORY_CONFIG);
+    EXPECT_EQ(tensor.get_logical_shape(), params.shape.logical_shape());
+}
+
+INSTANTIATE_TEST_SUITE_P(
+    EmptyTensorTestWithShape,
+    EmptyTensorTest,
+    ::testing::Values(
+        EmptyTensorParams{.shape=ttnn::Shape({1, 1, 32, 32})},
+        EmptyTensorParams{.shape=ttnn::Shape({2, 1, 32, 32})},
+        EmptyTensorParams{.shape=ttnn::Shape({64, 1, 256, 1})},
+        EmptyTensorParams{.shape=ttnn::Shape({1, 1, 21120, 16})},
+        EmptyTensorParams{.shape=ttnn::Shape({0, 0, 0, 0})},
+        EmptyTensorParams{.shape=ttnn::Shape({0, 1, 32, 32})},
+        EmptyTensorParams{.shape=ttnn::Shape({0})}
```

Comment:
wdyt about rank 0?

Reply:
Added tests for 1-volume tensors, including rank 0